schema/changelog-3.8.xml

```diff
@@ -8,6 +8,24 @@
 
   <changeSet author="author" id="changelog-3.8">
 
+    <createTable tableName="device_aliases">
```

Comment:
Did we decide on this name? It sounds confusing. I think we should rename it to `attribute_aliases` or something like that.